woocommerce/wc-admin
# Adding typescript resolver to fix linting issues with imports

No changelog

Just a quick add of a missing dependency for the typescript resolver. This was missing in a recent PR that changed our Eslint config to use the typescript resolver, the result being linting errors for all of our imports:

![image](https://user-images.githubusercontent.com/444632/111703853-7b4b3700-87fb-11eb-8f10-1a4c0a3f869c.png)

**Edit:**

I'm actually unsure of why so many of the wordpress packages are still unresolved since they _do_ appear in `package.json` dependencies:

![image](https://user-images.githubusercontent.com/444632/111706287-4345f300-87ff-11eb-98c4-21913e5cc70d.png)

Is this to be expected, @samueljseay ?

### Testing instructions

-   Checkout branch and run `npm install`
-   Open file in editor such as [this one](https://github.com/woocommerce/woocommerce-admin/blob/cda3b39d082624385a885132b2879db5d3685848/client/wp-admin-scripts/beta-features-tracking-modal/container.js#L0-L1) (I'm using vscode) and ensure that _most_ of the imports do **not** have a linting error. Currently some wordpress packages will still have unresolvable imports.
